File: python/unfinshed/vowelcount.py
# https://www.codewars.com/kata/54ff3102c1bad923760001f3/train/python
import numpy as np
import re

from numpy.lib import s_

def get_count(sentence):
    s = re.sub(r'[^a-zA-Z0-9]', '', lower(sentence))
    s_unique = np.unique(s, return_inverse = True)
    print(s_unique)
    np_array = np.array(list(s))

File: js/unfinished/vowel_count.js
//https://www.codewars.com/kata/54ff3102c1bad923760001f3/train/javascript

/*
Return the number (count) of vowels in the given string.
We will consider a, e, i, o, u as vowels for this Kata (but not y).
The input string will only consist of lower case letters and/or spaces.
*/

function getCount(str) {
  //var vowelsCount = 0;
  const regexVowels = /[aeiou]/i;
  // enter your magic here
  let vowelsCount = [...str].filter(function (value) {
    return value.match(regexVowels);
  });
  return vowelsCount.length;
}
